spec/handler/google_analytics_spec.rb

```diff
@@ -93,6 +121,15 @@ def env
     end
   end
 
+  describe "with user_id tracking" do
```

Comment:
the cookieDomain thingy is tested here too.
both can be tested again in the integration spec too.